internal/pkg/model/object.go

```diff
@@ -115,6 +118,7 @@ type Config struct {
 	ChangeDescription string                 `json:"changeDescription"`
 	Content           *orderedmap.OrderedMap `json:"configuration" validate:"required" diff:"true" configFile:"true"`
 	Blocks            Blocks                 `json:"-" validate:"dive" diff:"true"`
+	Orchestration     *Orchestration         `json:"-" validate:"omitempty,dive" diff:"true"`
```

Comment:
`Orchestration` ulozime do `model.Config`, bude ju mat vyplneny iba config z komponenty `keboola.orchestrator` (podobne ako `Blocks`):
- spravit `OrchestratorConfig`, ktory by embeddoval `Config` sice ide, ale na vsetkych miestach, kde sa ocakava `Conifg`, by som nemohol dat `OrchestratorConfig` -> tj. Go typovy system to takto neumoznuje (musel by sa spravit nejaky spolocny interface -> vela prace).
- rozmyslal som, spravit to cez `Relations`, ale to malo tiez vela negativ (`diff`, ...).
- takze som uprednostnil toto riesenie.

Reply:
👍